### PR TITLE
Remove citizen project creation notification

### PR DESCRIPTION
**Pull request by tony-tran**

### CODE REVIEW

File: tests/CitizenProject/CitizenProjectMessageNotifierTest.php [hunk 1]
```diff
@@ -29,12 +29,7 @@ public function testOnCitizenProjectApprove()
         $citizenProjectWasApprovedEvent = $this->createMock(CitizenProjectWasApprovedEvent::class);
         $committeeManager = $this->createMock(CommitteeManager::class);
         $router = $this->createMock(RouterInterface::class);
-
-        $producer->expects($this->once())->method('publish')->with(\GuzzleHttp\json_encode([
```
Comment (blazarecki) [inline tests/CitizenProject/CitizenProjectMessageNotifierTest.php]: The producer is still used in the app ?

Comment (tony-tran) [reply tests/CitizenProject/CitizenProjectMessageNotifierTest.php]: Nope

**Comment by thomasbisignani** [comment]
If the `scheduleCreationNotification` method is never used again, I think you can delete it ?

**Comment by tony-tran** [comment]
Yeah ! A oversight ! Thank you.

**Pull request merged**